src/modules/acbs/dto/acbs-get-bundle-information-response.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { DateString } from '@ukef/helpers';
+
+import { BundleAction } from './bundle-actions/bundle-action.type';
+
+export interface AcbsGetBundleInformationResponseDto<BundleMessageListItem extends BundleAction = BundleAction> {
+  PortfolioIdentifier: string;
+  // TODO APIM-125: Replace InitialBundleStatusCode with RequestedBundleStatus, as InitialBundleStatusCode is deprecated as of 01 dec 2024.
+  InitialBundleStatusCode: number;
```

Comment:
Is this something we can replace now to `RequestedBundleStatus`?

Reply:
I'll double check this with @avaitonis when he's back, as this TODO was copied from his ticket [APIM-114](https://ukef-dtfs.atlassian.net/browse/APIM-114) which was the POST version of the endpoint. Once I've got the okay from Audrius, I'm happy to replace this on both the GET and POST endpoints

Reply:
Ah, thanks Luke, I've changed this comment to `TODO APIM-311... `